resources/compass-runtime-agent/templates/deployment.yaml

```diff
@@ -19,15 +19,36 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "true"
+        container.apparmor.security.beta.kubernetes.io: runtime/default
       labels:
         app: {{ .Chart.Name }}
         release: {{ .Release.Name }}
     spec:
+      securityContext:
+        runAsUser: 65535
+        runAsGroup: 65535
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
       serviceAccountName: {{ .Chart.Name }}
       containers:
         - name: {{ .Chart.Name }}
+          securityContext:
+            privileged: false
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - ALL
+              add:
+                - NET_BIND_SERVICE
+            seLinuxOptions:
+              type: container_t
+            procMount: default
```

Comment:
I think this is also not needed as the default value is already good enough. @TorstenD-SAP

Reply:
I don't see a huge maintenance effort for this because there is nothing to decide or change. Just add it to the spec as it is. Therefor I vote for leaving it in.

Reply:
Thanks!